doc/source/data/doc_code/transforming_datasets.py

```diff
@@ -77,29 +77,80 @@ def transform_batch(df: pandas.DataFrame) -> pandas.DataFrame:
 # fmt: on
 
 # fmt: off
-# __writing_native_udfs_begin__
+# __writing_default_udfs_tabular_begin__
 import ray
 import pandas as pd
 
 # Load dataset.
 ds = ray.data.read_csv("example://iris.csv")
+ds.default_batch_format()
```

Comment:
Nit: Should we `print()` this, so if they're running this as a single script they will see the default batch format in the output?
```suggestion
print(ds.default_batch_format())
```